master/buildbot/steps/master.py

```diff
@@ -242,3 +246,37 @@ def start(self):
         self.addCompleteLog(name='Output', text=content)
         self.step_status.setText(self.describe(done=True))
         self.finished(SUCCESS)
+
+
+class CancelRelatedBuilds(BuildStep):
+    name = 'CancelRelatedBuilds'
+    description = ['Checking']
+    descriptionDone = ['Checked']
+
+    def __init__(self, isRelevant=None, builderNames=None, **kwargs):
+        BuildStep.__init__(self, **kwargs)
+
+        if isRelevant is None:
```

Comment:
That's the plan: to not only provide a step, but an example of how to use it out of the box.